### DISCUSSION TITLE
Facing the issue with Video Feed

### DISCUSSION THREAD

**Discussion by KavyashreeBasavesh** [ant-media/Ant-Media-Server#5694] [Q&A]
Building an application using Antmedia server and WebRTC.
The issue is, when user leaves the video conference call and rejoins, his video feed does not get displayed to other users.

@yashtandon113 @Mohit-3196 Please look into this issue. Dropped an email as well.

**Comment by burak-58** [comment]
Hi @KavyashreeBasavesh,
Thank you for the report. 
I want to understand the issue more to help you better.
Can I reproduce the issue by myself with sample pages? 
Please provide me a reproduce scenario.

**Reply by mekya** [reply]
Hi @burak-58,

Just a gentle reminder: I think @KavyashreeBasavesh already gives the reproduce scenario basically. Did you try it?
Secondly,  there is an email likely in our helpdesk as he mentioned. Have you had a chance to take a look?

Regards
Oguz

**Reply by tremzo** [reply]
@KavyashreeBasavesh I cannot find your email in the helpdesk. Whats your email address then I can locate it and support you. Thanks.

**Comment by tremzo** [comment]
@KavyashreeBasavesh I tried using multitrack-conference.html sample page, and could not reproduce the issue.

1. Join a room in 1 browser window
2. Join the same room using another browser window
3. Leave the room
4. Then rejoin and observe if the second user can see the participant that's rejoined. 

Could you elaborate on the reproduced scenario?

**Reply by lastpeony** [reply]
It is reproduceable on android. can you have a look at trackbasedconference activity

**Reply by mekya** [reply]
I think you've already created an issue for this @lastpeony? Please correct me if I'm wrong. 

If yes, could you please give the issue link here?

**Reply by lastpeony** [reply]
https://github.com/ant-media/Ant-Media-Server/issues/5737

**Comment by KavyashreeBasavesh** [comment]
Hi Team,

Please confirm any update on the issue being raised?

Regards,
Kavya

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by Mohit-3196** [comment]
Hi @KavyashreeBasavesh,
The GitHub issue https://github.com/ant-media/Ant-Media-Server/issues/5737 hasn't been assigned yet.
The team is completely occupied with pre-assigned issues and development works.

As my colleague mentioned earlier, we generally schedule issues according to priority, importance, and urgency. The pipeline is visible here
https://github.com/orgs/ant-media/projects/3

If you want us to increase the priority of any issue, you can reach out to [contact@antmedia.io](mailto:contact@antmedia.io)

Thank you.

**Comment by KavyashreeBasavesh** [comment]
Hi Team,

+++Looping  ***@***.***

for the help with the issue we are facing.

Regards,
Kavya

**Comment by KavyashreeBasavesh** [comment]
Hi Team,

Is there any progress?

Regards,
Kavya

**Comment by Mohit-3196** [comment]
Hi @KavyashreeBasavesh,
Pardon me but are you asking for updates from me?
If yes, please refer to my comment above.

> Hi @KavyashreeBasavesh,
The GitHub issue https://github.com/ant-media/Ant-Media-Server/issues/5737 hasn't been assigned yet.
The team is completely occupied with pre-assigned issues and development works.

As my colleague mentioned earlier, we generally schedule issues according to priority, importance, and urgency. The pipeline is visible here
https://github.com/orgs/ant-media/projects/3

If you want us to increase the priority of any issue, you can reach out to [contact@antmedia.io](mailto:contact@antmedia.io)

Thank you.

**Comment by KavyashreeBasavesh** [comment]
Hi Mohit,

I am referring to @***@***.*** for the support to prioritize this
issue.

Also, Mohit I just want to understand from last week you and your
colleague giving the same reason and not prioritizing even following up
with you guys. Atleast, you should consider our request and try to
prioritize instead of giving reasons.

Regards,
Kavya

**Reply by Mohit-3196** [reply]
Hi @KavyashreeBasavesh,
I understand your comment and have complete empathy with you.
And at the same time, I expect that you understand us as well. We always try to keep things open and transparent.
The pipeline is visible here https://github.com/orgs/ant-media/projects/3
And we are scheduling the tasks as per the priority, and urgency, and the availability of the team.

So if it is something very urgent for you, and you want us to change the pipeline and increase the priority of your task then kindly reach out to the solutions team at the above address so that we can make this win-win for all.

Thank you.
